Herbst 2020 Update: Kinder-Tarif entfernen, Arbeitsleistung präzisieren, uvm. (#6)
welcomewerkstatt/dokumente: beitragsordnung.md

# Beitragsordnung

Der Vorstand des Vereins Welcome Werkstatt e. V. hat folgende Beitragsordnung beschlossen:

1. Mitgliedsgebühren
    1. Alle Vereinsmitglieder – sowohl Fördermitglieder als auch ordentliche Mitglieder mit Stimmrecht – zahlen einen Mitgliedsbeitrag. Der Mitgliedsbeitrag wird monatlich berechnet.

>     2. Ordentliche Mitglieder können gemäß der Satzung § 6 einen Antrag auf Ermäßigung der Beitragshöhe stellen. Der formlose Antrag ist dabei zu begründen. Die Begründung kann sich z.B. auf den Erhalt von Sozialleistung im Rahmen des SGB I ff. stützen. Über den Antrag wird durch den Vorstand per einfachem Mehrheitsbeschluss binnen 14 Tagen entschieden. Der zu leistende Beitrag wird durch den Vorstand im Einzelfall festgelegt und dem Mitglied ohne Verzug formlos mitgeteilt.

Reviewer: ```suggestion
    2. Ordentliche Mitglieder können gemäß § 6 der Satzung einen Antrag auf Ermäßigung der Beitragshöhe stellen. Der formlose Antrag ist dabei zu begründen. Die Begründung kann sich z.B. auf den Erhalt von Sozialleistung im Rahmen des SGB I ff. stützen. Über den Antrag wird durch den Vorstand per einfachem Mehrheitsbeschluss binnen 14 Tagen entschieden. Der zu leistende Beitrag wird durch den Vorstand im Einzelfall festgelegt und dem Mitglied ohne Verzug formlos mitgeteilt.
```

    4. Der monatlich berechnete Beitrag beträgt:
        | Art der Mitgliedschaft  | Mitglieder mit Stimmrecht | Fördermitglieder |
        | ----------------------- | ------------------------- | ---------------- |
        | Erwachsene (ab vollendetem 18. Lebensjahr) | 25 Euro | Ab 12,50 Euro |
        | Partner/Familie (im gleichen Haushalt wie Mitglied) | 7,50 Euro | - |
        | Jugendliche (ab vollendetem 16. Lebensjahr) | 15 Euro | - |
        | Senioren (ab vollendetem 65. Lebensjahr) | 15 Euro | Ab 7,50 Euro |

        Der vierteljährliche, halbjährliche und jährliche Beitrag berechnet sich entsprechend durch Multiplikation der Monatsbeiträge mit der Anzahl der Monate (4, 6 oder 12).

2. Tagespauschale
    1. Für die private Nutzung der Räumlichkeiten des Vereins entrichten ordentliche Mitglieder mit Stimmrecht eine reduzierte Tagespauschale von 3 Euro. Gäste entrichten pro Nutzungstag innerhalb der regulären Öffnungszeiten pauschal 10 Euro.

>     2. Alle Besucher (Mitglieder und Gäste) können eine Begleitperson mit in die Räumlichkeiten bringen, sofern diese am gleichen Projekt mitarbeitet. Die Begleitperson eines Mitglieds entrichtet eine Tagespauschale von 3 Euro. Die Begleitperson eines Nicht-Mitglieds entrichtet eine Tagespauschale von 8 Euro.

Reviewer 2: Vorschlag von Thorben:

```suggestion
    2. Gäste, die Sozialleistung im Rahmen des SGB I ff. erhalten, zahlen pro Nutzungstag innerhalb der regulären Öffnungszeiten pauschal 5 Euro. Eine Kontrolle findet hierrüber nicht statt. Eine Mitteilung über die Inanspruchnahme, an die Aufsicht, ist ausreichend.
    
    3. Alle Besucher (Mitglieder und Gäste) können eine Begleitperson mit in die Räumlichkeiten bringen, sofern diese am gleichen Projekt mitarbeitet. Die Begleitperson eines Mitglieds entrichtet eine Tagespauschale von 3 Euro. Die Begleitperson eines Nicht-Mitglieds entrichtet eine Tagespauschale von 8 Euro.
```

Reviewer 2: Finde ich von der Idee her gut. Aber vielleicht kann man die letzten beiden Sätze einfach streichen. Es gibt ja bisher eh keine Kontrolle über gezahlte Tagespauschalen – egal ob mit oder ohne Sozialleistungen.

Author: Kann man gerne streichen. Dachte nur für Gäste, die sich informieren wollen ist es gut sowas zu lesen.

Reviewer 3: Ich finde schon gut, dass einmal aufgeschrieben ist, dass man keinen Nachweis darüber erbringen muss, dass Sozialleistungen gezahlt werden. Vllt. "Gäste müssen hierrüber keine Nachweis erbringen."

Reviewer 2: Sollten wir das eigentlich nicht auch für Mitglieder ermöglichen? Also wenn ein Mitglied den ermäßigten Monatstarif nutzt, wäre es doch auch schlüssig, wenn die Person dann auch nur 5€ Tagespauschale bezahlt, oder?

Author: Sicher ein guter Punkt. Aber du meinst 1,5 €? oder?

Author: Dann könnte man eher eine Art der Formulierung aufnehmen: "Gäste und Mitglieder, die Sozialleistung(-en) im Rahmen des SGB I ff. erhalten, zahlen pro Nutzungstag pauschal 50 % der normalen Tagespauschale. Gäste müssen hierrüber keinen Nachweis erbringen."

>     3. Mitglieder können zur Entrichtung der Tagespauschale eine Zehnerkarte zum Preis von zehn Tagespauschalen (aktuell 10 x 3 Euro also 30 Euro) erwerben. Sie erlaubt dem Mitglied die zehnmalige Nutzung der Räumlichkeiten – auch wenn sich die Höhe der Tagespauschale im Nachhinein ändert. 

Reviewer 2: Vorschlag von Thorben:
```suggestion
    4. Mitglieder können zur Entrichtung der Tagespauschale eine Zehnerkarte zum Preis von zehn Tagespauschalen (aktuell 10 x 3 Euro also 30 Euro) erwerben. Sie erlaubt dem Mitglied die zehnmalige Nutzung der Räumlichkeiten – auch wenn sich die Höhe der Tagespauschale im Nachhinein ändert.
    
    5. Mitglieder die Arbeitsleistungen für die Werkstatt erbringen (mind. eine Stunde) müssen am selben Tag für die private Nutzung keine Tagespauschale leisten. Dabei erfolgt eine Eintragung im Kassenbuch mit dem Vermerk "Arbeitsleistung". 
```

Reviewer 2: Bin mir unsicher bezüglich zwei Aspekten:
- "eine Stunde": Würde das etwas runterschrauben. 45 Minuten vielleicht?
- Eintrag im Kassenbuch: Würde ich weglassen. Einträge mit 0€ machen das Kassenbuch unübersichtlich. Finde es ausreichend an die Eigenverantwortung der Mitglieder zu appellieren.

Author: Finde eine Stunde als Festsetzung erstmal gut. Man kann das ja mit Augenmaß handhaben also auch mal aufrunden. Letzteres wäre für mich auch in Ordnung

Reviewer 3: Sollten wir noch spezifizieren, dass man das dann natürlich nicht noch zusätzlich als Arbeitsleistung für die Pflichtstunden einreichen kann?

4. Neumitglieder
    1. Der Verein erhebt eine Aufnahmegebühr von 10 Euro, die nach Aufnahme in den Verein fällig wird und mit dem ersten Mitgliedsbeitrag zusammen überwiesen werden kann oder gemeinsam mit dem ersten Mitgliedsbeitrag im Lastschriftverfahren eingezogen wird.

    2. Die Nutzung von Lagerfläche erfordert eine genormte Lagerkiste. Diese erhält das Mitglied zum Selbstkostenpreis von aktuell 8,50 Euro. Sofern die Kiste vollständig intakt und weiterhin nutzbar ist, kann diese bei Austritt an den Verein zurückgegeben werden und der Betrag wird dem Mitglied erstattet. 

    3. Auf Wunsch erhalten Mitglieder einen Schlüsselanhänger zur Öffnung der Räumlichkeiten. Dabei wird aktuell eine Pfandleistung in Höhe von 15 Euro fällig. Sofern der Schlüsselanhänger intakt und weiterhin nutzbar ist, kann dieser bei Austritt an den Verein zurückgegeben werden. Die gezahlte Pfandleistung wird dann vollständig erstattet.

5. Sonstiges
    1. Anschriften-, Namens- und Kontenänderungen sowie Veränderungen, die zu einer anderen Bemessung der Beitragshöhe führen, sind dem Vorstand unverzüglich mitzuteilen.

>     2. Wird der Beitrag trotz schriftlicher Mahnung nicht fristgemäß entrichtet, führt dies gemäß Satzung § 4 nach einem Verzug mit drei Monatsbeiträgen zum Ausschluss des Mitgliedes. Die Verbindlichkeiten bleiben davon unberührt. Für Mahnschreiben ist der Verein berechtigt, ohne Nachweisführung einen Betrag von 2,50 Euro zu berechnen.

Reviewer: ```suggestion
    2. Wird der Beitrag trotz schriftlicher Mahnung nicht fristgemäß entrichtet, führt dies gemäß  § 4 der Satzung nach einem Verzug mit drei Monatsbeiträgen zum Ausschluss des Mitgliedes. Die Verbindlichkeiten bleiben davon unberührt. Für Mahnschreiben ist der Verein berechtigt, ohne Nachweisführung einen Betrag von 2,50 Euro zu berechnen.
```

    3. Die Kontrolle erbrachter Arbeitsstunden obliegt dem Vorstand, dabei werden die Arbeitsstunden der Mitglieder durch den Vorstand oder den Mitgliederwart schriftlich bestätigt.

    4. Wird die Anzahl der Arbeitsstunden nicht erfüllt, erhebt der Verein pro nicht geleistete Stunde 10 Euro. Der eventuell fällig werdende Betrag wird dem Mitglied schriftlich mitgeteilt. Im folgenden Monat wird der Betrag durch das Mitglied in bar, per Lastschrift oder per Überweisung beglichen.

4. Neumitglieder
    1. Der Verein erhebt eine Aufnahmegebühr von 10 Euro, die nach Aufnahme in den Verein fällig wird und mit dem ersten Mitgliedsbeitrag zusammen überwiesen werden kann oder gemeinsam mit dem ersten Mitgliedsbeitrag im Lastschriftverfahren eingezogen wird.

    2. Die Nutzung von Lagerfläche erfordert eine genormte Lagerkiste. Diese erhält das Mitglied zum Selbstkostenpreis von der Werkstatt. Aktuell beträgt dieser 8,50 €. Bei Austritt ist es möglich die Kiste an den Verein zurück zu geben. Sofern die Kiste vollständig intakt und weiterhin nutzbar ist, wird der geleistete Betrag zurückerstattet.  

>     3. Für den Erhalt eines AirKey-Token wird eine Pfandleistung in Höhe des aktuellen Preises erhoben. Aktuell beträgt dieser 15 €. Sofern der Air-Key intakt und weiterhin nutzbar ist, kann dieser an den Verein zurückgegeben werden. Die Pfandleistung wird dann vollständig erstattet. 

Reviewer 2: Der Produkt/Eigenname der Schließanlage gehört hier nicht rein, imo.

```suggestion
    3. Auf Wunsch erhalten Mitglieder einen Schlüsselanhänger zur Öffnung der Räumlichkeiten. Dabei wird eine Pfandleistung in Höhe von 15 € fällig. Sofern der Schlüsselanhänger intakt und weiterhin nutzbar ist, kann dieser bei Austritt an den Verein zurückgegeben werden. Die Pfandleistung wird dann vollständig erstattet. 
```

Die Mitgliederversammlung des Vereins Welcome Werkstatt e.V. hat am 9.11.2019 folgende Beitragsordnung beschlossen:

1. Miedergebühren
1.1. Alle Vereinsmitglieder – sowohl Fördermitglieder als auch ordentliche Mitglieder mit Stimmrecht – zahlen einen Mitgliedsbeitrag. Der Mitgliedsbeitrag wird monatlich berechnet.

> 1.2. Ehrenmitglieder sind von der Beitragszahlung befreit. 

Reviewer 2: Damit aus diesem Format hier (Markdown) ordentlich eingerückte, nummerierte Listen werden, musst du die Sub-Aufzählungspunkte korrekt einrücken (4 Leerzeichen), und du musst auch die darüberliegende Nummerierung nicht wiederholen, also zum Beispiel:
```suggestion
    2. Ehrenmitglieder sind von der Beitragszahlung befreit. 
```

Reviewer 2: Siehe auch: https://www.markdownguide.org/basic-syntax/#ordered-lists

Author: Ich wollte gerne weg von der fortlaufenden Nummerierung hin zu einer dezimalen Gliederung mit Überschriften. Ich finde es dadurch deutlich Übersichtlicher.

Reviewer 2: > Ich wollte gerne weg von der fortlaufenden Nummerierung hin zu einer dezimalen Gliederung mit Überschriften. Ich finde es dadurch deutlich Übersichtlicher.

Das hab ich schon verstanden. Diese mehrstufige Untergliederung klappt in Markdown aber durch Einrückungen. Jede Einrückung um 4 Leerzeichen fügt eine Gliederungsebene hinzu. Wie das dargestellt wird (also ob dann da 1.1. oder 1. steht oder A. oder 1.A.) das können wir auf der Webseite immer noch modifizieren. Wichtig ist nur, dass es semantisch, also inhaltlich korrekt untergliedert ist.

Reviewer 2: Habe alle Einrückungen korrigiert.

> 1.2. Ehrenmitglieder sind von der Beitragszahlung befreit. 

Reviewer: Wer und wie kann man Ehrenmitglied werden? Davon steht nichts in der Satzung

Author: Das ist sicher ein guter Punkt. Wäre in der Satzung aufzunehmen, oder? Vllt. können wir den Punkt in der Beitragsordnung erstmal rausnehmen.

Reviewer: Ja, das müsste man dann erst in der Satzung klären. Ich wäre daher auch für aus der Beitragsordnung rausnehmen.

Author: Dann nimm es doch gerne direkt raus.

Reviewer 2: Ist raus.

> 1.3. In Sonderfällen können gemäß Satzung § 6 auch von ordentlichen Mitgliedern mit Stimmrecht begründete Anträge auf Ermäßigung der Beitragshöhe gestellt werden.

Reviewer: In § 6 der Satzung steht nichts von der Möglichkeit Anträge zu stellen, sondern nur, dass der Vorstand eine Einzelfallregelung per Mehrheitsbeschluss festlegen kann. 
Ich versteh leider nicht, was genau 1.3 regeln soll - dass man Anträge auf Ermäßigung stellen kann oder dass auch ordentliche Mitglieder ermäßigte Beiträge zahlen könnten?

Author: Jedes Mitglied könnte einen Antrag auf Ermäßigung stellen. Gemeint ist letztlich eine Ermäßigung des Beitrags falls ein Mitglied Arbeitslos wird oder ander Gründe vorliegen.

Reviewer: Dann kann der Punkt aus der Beitragsordnung raus, da er ja in der Satzung geregelt wird.

Author: Ich habe die Regelung nochmal etwas besser formuliert und würde den eigentlich gerne als Präzisierung drin lassen.

    | Kinder (bis zum vollendeten 14. Lebensjahr) | 7,50 Euro | - |

>     | Senioren (ab vollendetem 65. Lebensjahr) | 15 Euro | Ab 7,50 Euro |

Author: Ich würde Kinder rausnehmen und Jugendliche (ab 16 Jahren) aufnehmen. Dann könnte man noch einen zusätzlichen Punkt aufnehmen -> Zustimmung der Eltern betreffend.

>     | Senioren (ab vollendetem 65. Lebensjahr) | 15 Euro | Ab 7,50 Euro |

Reviewer 2: Ups ja, ich hatte auch die falschen Zeilen markiert. Senioren sollten nicht rausfliegen. Kinder entfernen und Jugendlich ab 16 finde ich gut!

>     | Senioren (ab vollendetem 65. Lebensjahr) | 15 Euro | Ab 7,50 Euro |

Reviewer 2: Habe die Kinder entfernt.

> Der Quartale bzw. (Halb-)Jährliche Beitrag berechnet sich entsprechend.

Reviewer 2: ```suggestion
Der quartale, halbjährliche und jährliche Beitrag berechnet sich entsprechend durch Multiplikation der Monatsbeiträge mit der Anzahl der Monate (4, 6 oder 12).
```

Author: Ich würde Kinder rausnehmen und Jugendliche (ab 16 Jahren) aufnehmen. Dann könnte man noch einen zusätzlichen Punkt aufnehmen -> Zustimmung der Eltern betreffend.

2. Tagessatz

> 2.1. Für die private Nutzung der Räumlichkeiten des Vereins entrichten ordentliche Mitglieder mit Stimmrecht eine reduzierte Tagespauschale von 3 Euro. Alle anderen Personen (Gäste) entrichten pro Nutzungstag innerhalb der regulären Öffnungszeiten pauschal mindestens 10 Euro.

Reviewer: "Mindestens 10 Euro" finde ich zu unkonkret 
```suggestion
2.1. Für die private Nutzung der Räumlichkeiten des Vereins entrichten ordentliche Mitglieder mit Stimmrecht eine reduzierte Tagespauschale von 3 Euro. Gäste entrichten pro Nutzungstag innerhalb der regulären Öffnungszeiten pauschal 10 Euro.
```

Author: Wir können das "mindestens" rausnehmen. Bisher stand es in der Beitragsordnung.

Reviewer 3: Muss hier nochmal auf das Stimmrecht verwiesen werden? Das ist doch durch ordentlich schon gegeben.

2.2. Alle Besucher (Mitglieder und Gäste) können eine Begleitperson mit in die Räumlichkeiten bringen, sofern diese am gleichen Projekt mitarbeitet. Die Begleitperson eines Mitglieds entrichtet eine Tagespauschale von 3 Euro. Die Begleitperson eines Nicht-Mitglieds entrichtet eine Tagespauschale von 8 Euro.

> 2.3 Den Mitgliedern steht die Möglichkeit einer 10er-Karte zur Verfügung. Diese ist ähnlich einen Gutschein nutzbar und eingenständig zu entwerten. Es ist Vorkasse (Bar oder Überweisung) erforderlich. Die Kosten bzw. der Wert der 10er-Karte entspricht zehn einzelnen Tagesätzen für Mitglieder gem. der Beitragsordnung.

Reviewer: ```suggestion
2.3 Den Mitgliedern steht zur Bezahlung der Tagespauschale die Möglichkeit der Nutzung einer 10er-Karte zur Verfügung. Diese ist ähnlich einem Gutschein nutzbar und eingenständig vor Ort zu entwerten. Es ist Vorkasse (Bar oder Überweisung) erforderlich. Die Kosten bzw. der Wert der 10er-Karte entspricht zehn Tagesätzen für Mitglieder gem. der Beitragsordnung.
```

Author: Ist nochmal auf der 10er Karte vermerkt. Im Prinzip ist die 10er Karte ein Gutschein im Wert von 30 €. Die 10er Karte verliert nicht ihren Wert durch eine Anpassung der Tagespauschale, sondern kann weiter eingelöst werden. Wie man dann im einzelnen damit umgeht müsste man schauen. Eine Anpassung steht aber ja auch nicht an, oder? Der Beitrag hat ja eher steuerliche Gründe

Reviewer: Da eine Erhöhung der Tagespauschale derzeit ja wirklich nicht ansteht, würde ich dieses potentielle Problem auch auf später, wenn es aktuell werden könnte, verschieben.

> 

Reviewer: Auch wenn es nicht wahrscheinlich ist, dass sich die Tagespauschale erhöht, würde ich noch ergänzen, was passieren soll, wenn sich die Tagespauschale erhöht, Mitglieder aber noch "Guthaben" auf der 10er Karte haben. Können dann die 10er Karten zurückgegeben werden und man erhält das Guthaben ausgezahlt? Oder gilt dann die bereits bezahlte, niedrigere Tagespauschale bis zum Verbrauch der 10er Karte weiter? Oder muss das Mitglied die Differenz zur höheren Tagespauschale nachzahlen?

Author: Habe das nochmal angepasst.

> 3. Arbeitsleistung

Reviewer: Problematisch finde ich noch, was genau "jährlich" bedeuten soll. Innerhalb eines Kalenderjahres oder ein Jahr ab Eintritt des Mitglieds? Ersteres ist bestimmt leichter zu überblicken - andererseits müssten dann Mitglieder, die im Dezember eintreten, entweder noch schnell die 5 Arbeitsstunden leisten oder sie müssten gleich zu Beginn 50 € zahlen - das finde ich persönlich etwas unverhältnismäßig. Ein Jahr ab Eintritt wäre aber wohl etwas mühsam, das wirklich zu überprüfen

Author: Ich würde folgende Alternativen vorschlagen: 1. Es gilt erst ab dem 1. vollen Kalenderjahr eines Mitglieds und wird dann dem Kalenderjahr entsprechend abgerechnet. 2. Es gilt anteilig für das Eintrittsjahr und wird dann im Kalenderjahr abgrechnet.

> 3.1. Alle ordentlichen Mitglieder mit Stimmrecht ab dem vollendeten 16. Lebensjahr und bis zu vollendeten 60. Lebensjahr müssen jährlich 5 Stunden Arbeit zum Erhalt, zur Pflege und/oder Erweiterung der vereinseigenen Einrichtungen und Anlagen erbringen. 

Reviewer 2: Vertippt? Oder wieso 5?

```suggestion
3.1. Alle ordentlichen Mitglieder mit Stimmrecht ab dem vollendeten 16. Lebensjahr und bis zu vollendeten 60. Lebensjahr müssen jährlich 4 Stunden Arbeit zum Erhalt, zur Pflege und/oder Erweiterung der vereinseigenen Einrichtungen und Anlagen erbringen. 
```

Author: Ich habe die Stunden mal ausgerechnet

Offene Werkstatt 26 x 4 Std. = 96 Std. 
Repair Café ca. 6 x 6 Std. = 36 Std. 
Aufräumen ca. 4 x 5 Std. = 20 Std. 
= 152 Std. / 50 Mitglieder = 3 Std. 

Darin ist kein Ausbau oder ähnliches involviert. Auch die Kurse noch nicht. Daher wollte ich 5 vorschlagen. Ist sicher ein diskutabler Punkt.

Author: Könnten auch nochmal darüber nachdenken wie man mit der Tagespauschale umgeht, wenn jemand für die Werkstatt Arbeitsleistung erbringt. Evtl. so etwas wie: "An Tagen an denen ein Mitglied Arbeitsleistungen für die Werkstatt erbringt ist keine Tagespauschale zu leisten."

Reviewer 2: Bin nach wie vor für 4 Stunden – weil das der Länge einer Aufsicht und unserer sonstigen Veranstaltungen entspricht und sich damit gut argumentieren lässt, dass man nur einmal im Jahr einen solchen Dienst übernehmen muss.

Author: Der Punkt ist sicher richtig. Aber genau das würde ich erweitern wollen. Also das jedes Mitglied müsste damit eine Aufsicht übernehmen und hilf nochmal eine Stunde beim Ausbau, der Pflege oder ähnlichem.

Reviewer 2: Haben uns auf 4 geeinigt.

> 3.2 Die Arbeitsstunden können dabei aus der Aufsicht der offenen Werkstatt, Kursleiter, Repair-Café Helfer, Ausbau und Pflege der Werksttat bestehen. 

Reviewer: ```suggestion
3.2 Die Arbeitsstunden können dabei in Form der Aufsicht der offenen Werkstatt, als Kursleiter oder Repair-Café Helfer oder für den Ausbau und die Pflege der Werksttat abgeleistet werden. 
```

Author: Habe ich angepasst

> 3.2 Kontrolle erbrachter Arbeitsstunden obliegt dem Vorstand, dabei werden die Arbeitsstunden einzelner Mitglieder durch den Vorstand oder den Mitgliederwart schriftlich bestätigt.

Reviewer: ```suggestion
3.2 Die Kontrolle erbrachter Arbeitsstunden obliegt dem Vorstand, dabei werden die Arbeitsstunden der Mitglieder durch den Vorstand oder den Mitgliederwart schriftlich bestätigt.
```

Author: Habe ich angepasst

> 3.3 Wird die Anzahl der Arbeitsstunden nicht erfüllt, erhebt der Verein pro nicht geleistete Stunde 10 Euro. Der eventuell fällig werdende Betrag wird dem Mitglied schriftlich mitgeteilt. Im folgenden Monat wird der Betrag durch das Mitglied in bar, per Lastschrift oder per Überweisung beglichen.

Reviewer: ```suggestion
3.3 Wird die Anzahl der Arbeitsstunden nicht erfüllt, erhebt der Verein pro nicht geleisteter Stunde 10 Euro. Der eventuell fällig werdende Betrag wird dem Mitglied schriftlich mitgeteilt. Im folgenden Monat wird der Betrag durch das Mitglied in bar, per Lastschrift oder per Überweisung beglichen.
```

Schriftlich = per Brief. Reicht nicht zwecks Praktikabilität auch per E-Mail?

Author: Gem. § 126 BGB darf die schriftliche Form (Brief), doch sowieso durch die elektronische Form (E-Mail) ersetzt werden wenn nicht anders vorgeschrieben. Also ich würde vorschlagen eine Excel Liste in der Cloud zu führen und die Mitglieder müssten sich beim Vorstand melden und bekommen dann ein Bestätigung.

Reviewer: Elektronische Form meint aber (gem. § 126a BGB) mit einer qualifizierten elektronischen Signatur, das macht es dann etwas komplizierter. Ich würde daher einfach aufnehmen "per E-Mail" statt "schriftlich"

Author: Änderst du es dann in dem Entwurf? Dann können wir diesen Punkt schließen.

4. Neumitglieder
4.1. Der Verein erhebt eine Aufnahmegebühr von 10 Euro, die nach Aufnahme in den Verein fällig wird und mit dem ersten Mitgliedsbeitrag zusammen überwiesen werden kann oder gemeinsam mit dem ersten Mitgliedsbeitrag im Lastschriftverfahren eingezogen wird.

> 4.2 Nutzung von Lagerfläche erfodert eine genormte Lagerkiste. Diese erhält das Mitglied zum Selbstkostenpreis von der Werkstatt.Aktuell beträgt dieser 8,50 € 

Reviewer 2: ```suggestion
4.2 Nutzung von Lagerfläche erfodert eine genormte Lagerkiste. Diese erhält das Mitglied zum Selbstkostenpreis von der Werkstatt. Aktuell beträgt dieser 8,50 € 
```

Reviewer 3: Können wir den Preis nicht einfach weg lassen? Dann müssen wir uns nicht darum kümmern das regelmäßig zu aktualisieren.

Author: Können wir. Sollten dann nur eine Alternative finden, wie: "Der aktuelle Preis wird im Rahmen des Neumitgliederprozesses bekannt gegeben."

> 4.2 Nutzung von Lagerfläche erfodert eine genormte Lagerkiste. Diese erhält das Mitglied zum Selbstkostenpreis von der Werkstatt.Aktuell beträgt dieser 8,50 € 

Reviewer: ```suggestion
4.2 Die Nutzung von Lagerfläche erfordert eine genormte Lagerkiste. Diese erhält das Mitglied zum Selbstkostenpreis von der Werkstatt. Aktuell beträgt dieser 8,50 € 
```

Reviewer 2: Done

> 4.3 Für den Erhalt eines AirKey-Token wird eine Pfandleistung in Höhe des aktuellen Preises erhoben. Aktuell beträgt dieser 15 €

Reviewer 2: Nicht sicher ob wir das so regeln wollen. Die AirKey-Anhänger nutzen sich ja auch ab. Würden wir dann neuen Mitgliedern einen gebrauchten Anhänger ebenfalls wieder für 15€ anbieten?

```suggestion
4.3 Für den Erhalt eines AirKey-Schlüsselanhängers wird eine Pfandleistung in Höhe des aktuellen Preises erhoben. Aktuell beträgt dieser 15 €.
```

Reviewer 3: Ich würde sagen ja, Bei dem Pfand müssen wir ja eigentlich jeweils den Neuanschaffungspreis im Auge behalten.

5. Zahlungsweise
5.1 Bei Nutzung des Lastschriftverfahrens und Unregelmäßigkeiten auf den Konten des Vereinsmitglieds – z.B. mangelnde Deckung, Änderung der Kontonummer, der Anschrift, des Namens u.a. ohne Information an den Verein – werden pro Mahnung etwaige Rücklastschriftgebühren und eine Mahngebühr von 2,50 Euro erhoben.

> 5.2 Der Mitgliedsbeitrag kann auch per Überweisung entrichtet werden. Barzahlung des Mitgliedsbeitrags ist ausgeschlossen. 

Reviewer: Die Zahlung des Mitgliedsbeitrages ist schon unter Nr. 1 geregelt.

Author: Habe ich angepasst

5.3. Es können Umlagen und / oder Sachleistungen von den Mitgliedern erhoben werden. Die Erhebung von Umlagen und / oder Sachleistungen muss von der Mitgliederversammlung beschlossen werden.

6. Sonstiges
6.1. Anschriften-, Namens- und Kontenänderungen sowie Veränderungen, die zu einer anderen Bemessung der Beitragshöhe führen, sind dem Vorstand unverzüglich mitzuteilen.

> 6.2. Wird der Beitrag trotz schriftlicher Mahnung nicht fristgemäß entrichtet, führt dies gemäß Satzung § 4 nach einem Verzug mit drei Monatsbeiträgen zum Ausschluss des Mitgliedes. Die Verbindlichkeiten bleiben davon unberührt. Für einfache Mahnschreiben ist der Verein berechtigt, ohne Nachweisführung einen Betrag von 2,50 Euro zu berechnen.

Reviewer: Eine Voraussetzung für den Ausschluss eines Mitglieds kann m.E. nicht in der Beitragsordnung geregelt werden - das müsste in die Satzung.

Author: Es steht bereits unter § 4 was in der Satzung. Die Beitragsordnung konkretisiert hier scheinbar. In §4 der Satzung steht "Die Mitgliedschaft endet: 4. bei Mitgliedern, die sich mit mehr als drei Monatsbeiträgen im Verzug befinden, automatisch."

Reviewer: Ah, das hatte ich übersehen, dass das schon so in der Satzung geregelt ist. Dann finde ich es gut so 👍
